featuretools/primitives/standard/aggregation_primitives.py

```diff
@@ -289,8 +289,8 @@ def chunk(s):
             def format_chunk(x):
                 return x[:].fillna(0)
 
-            return (s.agg(lambda x: format_chunk(x).sum()),
-                    s.agg(lambda x: len(format_chunk(x))))
+            return (s.agg(lambda x: format_chunk(x).sum()).astype('float64'),
+                    s.agg(lambda x: len(format_chunk(x))).astype('int64'))
```

Comment:
Ran into an issue where if the sum or the length was either 1 or 0 Pandas would convert the result to a boolean, which would cause errors down the line.